regexes/client/browsers.yml

```diff
@@ -5,6 +5,18 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+# Yaani Browser
+- regex: '(?:Turkcell-)?YaaniBrowser(?:/(\d+[\.\d]+))? Mobile'
+  name: 'Yaani Browser'
+  version: '$1'
+  engine:
+    default: 'Gecko'
```

Comment:
This expression will not match the type of browser engine for MacOS
`Mozilla/5.0 (iPhone; CPU iPhone OS 13_3_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Turkcell-YaaniBrowser/6.0.0 Mobile/15E148 Safari/605.1.15 Turkcell-TR`

I think it's better to replace with:
`Firefox/.*(?:Turkcell-)?YaaniBrowser(?:/(\d+[\.\d]+))?`